### PR TITLE
PLANNER-2753 Abandon optaweb-employee-rostering

### PR DESCRIPTION
**Pull request by triceo**
Upstream PR: https://github.com/kiegroup/optaplanner/pull/2084

### CODE REVIEW

**Review by rsynek — APPROVED** [review]
I think we can also remove https://github.com/kiegroup/optaweb-employee-rostering/tree/main/.github.

File: README.adoc [hunk 1]
```diff
@@ -1,28 +1,17 @@
 :projectKey: optaweb-employee-rostering
-:sonarBadge: image:https://sonarcloud.io/api/project_badges/measure?project={projectKey}
-:sonarLink: link="https://sonarcloud.io/dashboard?id={projectKey}"
+
+*This project is no longer maintained.*
+Visit https://github.com/kiegroup/optaplanner-quickstarts/[OptaPlanner Quickstarts] to see how to integrate https://www.optaplanner.org/[OptaPlanner] in your application.
```
Comment (Christopher-Chianelli) [inline README.adoc:4]: Maybe direct them directly to the employee scheduling quickstart (https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/employee-scheduling )

**Review by Christopher-Chianelli — APPROVED** [review]

**Comment by psingh-neudesic** [comment]
@triceo can you please throw some light on why is this project being abandoned? We have been building on top of this for a client.

**Comment by triceo** [comment]
@psingh-neudesic This project has not been actively maintained for a long time, other than the occasional version bump. We believe that, as a demo application, it has outlived its usefulness and we will continue spending our energy on the quickstarts.

The fact that this repo will no longer be maintained does not affect you or your clients in any way. You can still use this code and expand on it in any way permitted by the Apache License.

**Pull request merged**